packages/design/src/index.ts

```diff
@@ -111,8 +111,8 @@ function buildSVGStyle(
   const iconSizeStyle = iconSizes.tokens[size];
   const svgStyle: {
     fill?: string;
-    width: string | number;
-    height: string | number;
+    width: number;
```

Comment:
Checking the compiled output for design it is the exact same except for this but there are no typescript errors caused by this change